src/Intl/Idn/Idn.php

```diff
@@ -346,6 +342,18 @@ private static function process($domain, array $options, Info $info)
             $validationOptions = $options;
 
             if ('xn--' === substr($label, 0, 4)) {
+                // Step 4.1. If the label contains any non-ASCII code point (i.e., a code point greater than U+007F),
+                // record that there was an error, and continue with the next label.
+                if (1 === preg_match('/[^\x00-\x7F]/', $label)) {
+                    $info->errors |= self::ERROR_PUNYCODE;
+
+                    continue;
+                }
+
+                // Step 4.2. Attempt to convert the rest of the label to Unicode according to Punycode [RFC3492]. If
+                // that conversion fails and if not IgnoreInvalidPunycode, record that there was an error, and continue
```

Comment:
The reference to `IgnoreInvalidPunycode` should be removed

Reply:
Thanks for catching that